actions/plan/plan.sh

```diff
@@ -55,6 +47,14 @@ echo "##[group]terraform init"
 terraform init -input=false -backend-config="${BACKEND_CONFIG}"
 echo "##[endgroup]"
 
+git add --intent-to-add .terraform.lock.hcl
```

Comment:
TIL ``-intent-to-add``, in this case, it seems to be so we can generate a diff for what could possibly be an untracked file?